returnn/tf/network.py

```diff
@@ -1347,6 +1347,7 @@ def _make_layer(layer_cls, layer_dict, map_opts=True):
         opts = nest.map_structure(_map_layer_dict_value, opts)
       opts.pop("output", None)
       opts["output"] = layer_cls.get_out_data_from_opts(**opts)
+      opts["output"] = layer_cls.fixup_out_data(**opts, verify_shape=False)
```

Comment:
Why do you need `verify_shape`? This should never be needed, unless sth is wrong.

Reply:
Was just about to write a reply, see below